README.md

```diff
@@ -2,135 +2,137 @@
 
 ![madness](https://cloud.githubusercontent.com/assets/8445924/10410421/ccca8b24-6f30-11e5-9952-9e5be5c4d792.png)
 
-Manul is the madness dependencies vendoring utility for Golang programs.
+Manul is the dependency vendoring utility for Golang programs.
 
-## Why yet another utility
+## What the reason for yet another utility?
```

Comment:
What the reason for it? I guess, you're confusing this case with ending titles with period, which should not be done.